hail/src/main/scala/is/hail/io/fs/HadoopFS.scala

```diff
@@ -76,15 +77,15 @@ class HadoopFS(private[this] var conf: SerializableHadoopConfiguration) extends
     conf = _conf.asInstanceOf[SerializableHadoopConfiguration]
   }
 
-  def createNoCompression(filename: String): PositionedDataOutputStream = {
+  def createNoCompression(filename: String): PositionedDataOutputStream = retryTransientErrors {
     val fs = getFileSystem(filename)
     val hPath = new hadoop.fs.Path(filename)
     val os = fs.create(hPath)
     new WrappedPositionedDataOutputStream(
       HadoopFS.toPositionedOutputStream(os))
   }
 
-  def openNoCompression(filename: String): SeekableDataInputStream = {
```

Comment:
If the streams hiccup we'll still get errors, but this PR is a worthwhile improvement. I'm not sure how to robustly retry the input streams we get from Hadoop anyway.